app/helpers/completeness_helper.py

```diff
@@ -0,0 +1,26 @@
+from flask import g
+from flask_login import current_user, login_required
+from werkzeug.local import LocalProxy
+
+from app.globals import get_answer_store, get_metadata, get_completed_blocks
+from app.helpers.path_finder_helper import path_finder
+from app.questionnaire.completeness import Completeness
+
+
+@login_required
+def get_completeness():
```

Comment:
I think this should live in `globals.py` as that is the place we currently house this sort of logic. 
The "helpers" are currently more utility methods or wrappers

Reply:
This could even be made as a wrapper eg `path_finder_helper.full_routing_path_required`??

Reply:
Probably makes more sense as a function in `globals`